### ISSUE TITLE
Doctor Gregory Victor won't teach bandage recipes

### ISSUE THREAD

**Issue by Sardonic** [dalaranwow/dalaran-wow#5906]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
At First Aid level 240 and character level 61, I completed the quest Triage in Hammerfall, Arathi Highlands. I was rewarded with 20x Heavy Runecloth Bandage, but further speaking with Doctor Gregory Victor, **he does not teach any First Aid recipes**, even though he should offer the Heavy Mageweave Bandage, Runecloth Bandage, and Heavy Runecloth Bandage recipes.

In addition, his speech text is incorrect, appearing to be that of the equivalent questgiver in the Alliance:

> Greetings, Etiline. I am Doctor Gustaf VanHowzen, head trauma surgeon of Alliance Trauma.

**Expected behaviour**: Tell us what should happen instead
After completing the Triage quest, Doctor Gregory Victory should be able to teach the player the recipes Heavy Mageweave Bandage, Runecloth Bandage, and Heavy Runecloth Bandage, and should have speech that is accurate to his character.

**Steps to reproduce the problem**:

1. Complete the quest Triage as a Horde character
2. Speak to Doctor Gregory Victor
3. Notice that he uses the wrong speech and does not have an option to teach the player First Aid recipes

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/4325000/85935154-586aa280-b8bb-11ea-9ca8-4ab82dc9e828.png)

Notice that the General chat log demonstrates that I have completed Triage and received the appropriate rewards.

Comments here all vouch that he should teach these recipes after completing this quest: https://classic.wowhead.com/item=8545/heavy-mageweave-bandage#comments
https://classic.wowhead.com/spell=18629/runecloth-bandage#comments
https://classic.wowhead.com/spell=18630/heavy-runecloth-bandage#comments

The first-aid guide on the Dalaran-Wow forums back up these claims:
https://www.dalaran-wow.com/forums/input-and-assistance/guides-and-tutorials/topic/2643/first-aid-1-450

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Doctor Gregory Victor: npc=12920
Triage: quest=6622
Heavy Mageweave Bandage: item=8545

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
